src/components/articles/NewArticleModal.vue

```diff
@@ -1,8 +1,12 @@
 <template>
   <q-dialog v-model="promptModel" persistent>
     <q-card style="min-width: 350px; margin-top: -15%;" >
```

Comment:
Maybe placement of the prop could be in a different place? Thus avoid -15% style change